pc/registration/icp/rodrigues.go

```diff
@@ -11,9 +11,9 @@ import (
 func rodriguesToRotation(v mat.Vec3) mat.Mat4 {
 	ang := v.Norm()
 	r := mat.Mat4{
-		0, -v[2], v[1], 0,
-		v[2], 0, -v[0], 0,
-		-v[1], v[0], 0, 0,
+		0, v[2], -v[1], 0,
+		-v[2], 0, v[0], 0,
+		v[1], -v[0], 0, 0,
```

Comment:
Is this also a workaround the problem with `mat.Rotate`?

Reply:
It seemed correct before according to https://mathworld.wolfram.com/RodriguesRotationFormula.html for example.

Reply:
I'm adding a comment to `Mat4`:
```
// Mat4 represents 4x4 matrix stored by (column * 4 + row) index.
// (Column is stored first to optimize memory access when transforming vector.)
type Mat4 [16]float32
```